snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs

```diff
@@ -1,131 +1,3 @@
-//using System;
-//using System.Runtime.InteropServices;
-//using System.Text;
-
-
-//<Snippet2>
```

Comment:
Not related to the mentioned PRs, but I also didn't find usings of this snippet.

Reply:
@pkulikov, I'm finding that this file is still in use:

- docs\csharp\programming-guide\unsafe-code-pointers\pointer-conversions.md: [!code-csharp[csProgGuidePointers#4](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs#4)]
- docs\csharp\programming-guide\xmldoc\processing-the-xml-file.md:[!code-csharp[csProgGuidePointers#21](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs#21)]

Reply:
@rpetrusha yes, I've found those two usages. That's why the whole file *is not being removed*. Only  not used parts of it.

Reply:
Sorry, @pkulikov -- I should have noted that the entire file wasn't deleted!